lib/kbsecret/cli/kbsecret-session

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require "kbsecret"
+
+include KBSecret
+
+cmd = CLI.create do |c|
+  c.slop cmds: %w[new rm] do |o|
+    o.banner = <<~HELP
+      Usage:
+        kbsecret session [options] <new|rm> <label>
+    HELP
+
+    o.string "-t", "--team", "the team to create the session under"
+    o.array "-u", "--users", "the keybase users", default: [Keybase::Local.current_user]
+    o.string "-r", "--root", "the secret root directory"
+    o.bool "-f", "--force", "force creation (ignore overwrites, etc.)"
+    o.bool "-n", "--no-notify", "do not send a notification to session members"
+    o.bool "-d", "--delete", "unlink the session in addition to deconfiguration"
+  end
+
+  c.dreck do
+    string :command
+    string :label
+  end
+
+  c.ensure_session! :argument if c.args[:command] == "rm"
+end
+
+session_label = cmd.args[:label]
+
+case cmd.args[:command]
+when "new"
+  new_session session_label, cmd
+when "rm"
+  rm_session session_label, cmd
+else
+  cmd.die "Unknown subcommand: '#{cmd.args[:command]}'."
+end
+
+def new_session(label, cmd)
```

Comment:
Methods need to be declared above their usage -- put these declarations above `CLI.create` but below `include KBSecret`.